frontend/packages/core/src/Resolver/input.tsx

```diff
@@ -14,18 +14,31 @@ import {
 import { Button } from "../button";
 import { Error } from "../Feedback";
 import { TextField } from "../Input/text-field";
+import { client } from "../network";
 
 import type { ChangeEventTarget } from "./hydrator";
 import { convertChangeEvent, hydrateField } from "./hydrator";
 
 const Form = styled.form({});
 
 interface QueryResolverProps {
+  // The inputType is the orignal resource type requested
+  // eg: clutch.aws.ec2.v1.AutoscalingGroup
+  inputType: string;
```

Comment:
Typescript can pick this up as a docstring if you have it in the format
```suggestion
  /**
   * The inputType is the orignal resource type requested
   * eg: clutch.aws.ec2.v1.AutoscalingGroup
   */
  inputType: string;
```